data/pizza.json

```diff
@@ -1180,7 +1168,7 @@
 			"hidden_ingredients": [
 
```

Comment:
[Should pasta be a hidden ingredient?](https://sannegarden.com/meny/johanneberg/#:~:text=Bacon%2C%20%C3%84ggula%2C%20Svartpeppar%2C%20Gr%C3%A4dds%C3%A5s)  
Goes for all the pasta dishes.